frontend/webservice/entity.cgi

```diff
@@ -352,6 +364,21 @@ sub get_root_entities{
     return {results => \@entities};
 }
 
+sub get_entities{
+    my $method = shift;
+    my $params = shift;
+    my $ref = shift;
+
+    my $entities = OESS::DB::Entity::get_root_entities(db => $db);
```

Comment:
don't we want this to go deeper... like create a flat copy of all entities... and just their name/id not the entire object?